<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetHostInfoDiaog</class>
 <widget class="QDialog" name="SetHostInfoDiaog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetHostInfoDiaog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cEditAddress">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cEditPort">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cBtnOK">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cBtnCancle">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
